.gitignore

```diff
@@ -5,3 +5,4 @@ source/*/*.min.js.map
 test/results
 /dev.sh
 .DS_Store
+.idea
```

Comment:
You could have a personal gitignore outside of the project

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore